### PR TITLE
Update swedish translations

### PR DESCRIPTION
**Pull request by kernehed**
I hope I did it the right way.

### CODE REVIEW

File: custom_components/plex_assistant/localize.py [hunk 1]
```diff
@@ -121,6 +121,110 @@
             ],
         },
     },
+    "sv": {
+        "play": "spela",
+        "on_the": "på",
+        "not_found": "not found",
```
Comment (maykar) [inline custom_components/plex_assistant/localize.py]: "not_found" is used for errors like "episode not found" or "cast device not found". Should this be translated too?

Comment (kernehed) [reply custom_components/plex_assistant/localize.py]:
```suggestion
        "not_found": "finns ej",
```

Would be the right translation.

File: custom_components/plex_assistant/localize.py [hunk 2]
```diff
@@ -121,6 +121,110 @@
...
+            ],
+        },
+        "latest": {
+            "keywords": [
+                "senaste",
+                "nyligen",
+                "ny",
+            ],
+            "pre": [
+                "the",
```
Comment (maykar) [inline custom_components/plex_assistant/localize.py]: I think you may have missed this one. Using Google translate on the phrase "play the latest episode of" I get "spela det senaste avsnittet av". So, the post words seem good here, should the pre word for this be "det" and should that also be applied to unwatched, ondeck, season, and episodes also?

**Comment by maykar** [comment]
Would you also consider translating the README_SV.md file as well? Thank you for the work, I really appreciate it and I'm sure Swedish speaking users do as well!

File: custom_components/plex_assistant/localize.py (hunk 2 above)
Comment (kernehed) [reply custom_components/plex_assistant/localize.py]:
```suggestion
                "det",
```

You are totally right, it should be "det".

Comment (maykar) [reply custom_components/plex_assistant/localize.py]: Would it make sense to add this "pre" to other things like unwatched, ondeck, season, and episodes or is that not something that would usually be said?

Comment (kernehed) [reply custom_components/plex_assistant/localize.py]: Hmm, I don't think that will work,

It would be two different "pre". **Den/Det**.

Comment (maykar) [reply custom_components/plex_assistant/localize.py]: So should the pre for all be:
```
            "pre": [
                "den",
                "det",
             ]
```
or should just den or det be applied to certain ones?

Comment (maykar) [reply custom_components/plex_assistant/localize.py]: If you think those would need added we can change it, but in my testing on google translate it seems like neither det nor den would be used for the other options, but I'm not sure if google translate can really be trusted :) .

File: custom_components/plex_assistant/localize.py [hunk 3]
```diff
@@ -121,6 +121,110 @@
...
+            "post": [
+                "avsnitt",
+                "av",
+            ],
+        },
+        "ondeck": {
+            "keywords": [
+                "on deck",
+                "ondeck",
+            ],
```
Comment (maykar) [inline custom_components/plex_assistant/localize.py]: Should "on deck" be translated as well?

**Comment by maykar** [comment]
Going to pull this with the next release coming soon. Still not sure if "on deck" needs translated or not, but after pull it can be updated if need be.

**Pull request merged**